### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_Screenreader : Screen Reader is incorrectly announcing different level heading at same level.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/azuredatastudio#11199]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#MAS1.3.1;#SQL Azure Data Studio;#FTP;#NVDA;#JAWS;#Benchmark;#STP;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: 2d989e1
Date: 2020-07-03T05:33:38.245Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References: MAS1.3.1
Screenreader : NVDA 2019.3.1,JAWS

**Repro Steps:**
1.Launch Azure Data Studio Insiders application.
2.Connect to server.
3.Right-click connection and click New Notebook4. Open the Command Palette by pressing CTRL+SHIFT+P (Windows)
appears.
4.Type “Python” into the text box that appears. Select “Configure Python for Notebooks” from the menu.
5.Python Dependencies Wizard will appear.
6.Provide any Input file details & Next.
7.Start Screenreader and observe how NVDA is announcing the heading step 2 and Install Dependencies.

**Actual:**
NVDA is incorrectly announcing the heading step 2 and Install Dependencies at same heading level as heading level 1.

**Same issue also observed with JAWS.**

**Expected:**
NVDA should announce the heading step 2 as heading level 1 and Install Dependencies as heading level 2.

**User Impact:**
Screen reader users will not get page heading structure properly and get confused. 

**Attachment link for Reference:**

[11199_A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4869700/11199_A11y_AzureDataStudioStayHealthy_Notebook_Python.Dependencies.Wizard_Screenreader.zip)

**Comment by Rama8345** [comment]
Verified the issue at Azure Data Studio. the issue is not reproducible. So, closing the bug. Please find attachment for reference.
**Github Tags:**#Closed;
![11199_Closed](https://user-images.githubusercontent.com/115216656/221836633-8e106a52-c50c-4d80-8782-c7859c849b25.gif)
